beginner_source/export_simple_model_to_onnx_tutorial.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+"""
+Export a PyTorch model to ONNX
+==============================
+
+**Author**: `Thiago Crepaldi <https://github.com/thiagocrepaldi>`_
+
+.. Note::
+    As of PyTorch 2.1, there are two versions of ONNX Exporter.
+
+    * ``torch.onnx.dynamo_export` is the latest and recommended exporter basedon the TorchDynamo and is the default starting from PyTorch 2.1
```

Comment:
rephrased to

```python
.. Note::
    As of PyTorch 2.1, there are two versions of ONNX Exporter.

    * ``torch.onnx.dynamo_export`is the newest (still in beta) exporter based on the TorchDynamo technology released with PyTorch 2.0
    * ``torch.onnx.export`` is based on TorchScript backend and has been available since 1.2.0
```